packages/themes/dekode-theme/inc/assets.php

```diff
@@ -11,28 +11,57 @@
 /**
  * Hooks
  */
-\add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\scripts_and_styles' );
+\add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\do_enqueue_assets' );
+\add_action( 'enqueue_block_editor_assets', __NAMESPACE__ . '\\do_enqueue_block_editor_assets' );
 
 /**
- * Enqueue scripts and styles
+ * Enqueue front end styles and scripts.
  *
  * @return void
  */
-function scripts_and_styles() : void {
-	/**
-	 * Stylesheets
-	 */
+function do_enqueue_assets() : void {
+
+	// Enqueue styles.
 	$style_file_path = \get_template_directory() . '/build/style.css';
+	$style_file_uri  = \get_template_directory_uri() . '/build/style.css';
+
 	if ( file_exists( $style_file_path ) ) {
-		\wp_enqueue_style( 'dekode-theme-style', \get_template_directory_uri() . '/build/style.css', [], \wp_get_theme()->get( 'Version' ) );
+		\wp_enqueue_style( 'dekode-theme', $style_file_uri, [], filemtime( $style_file_path ) );
```

Comment:
We should avoid using filemtime. The function can be cached. Do we have any alternatives? For production I guess we could use a fixed version number, but that is just annoying when working locally.

Reply:
We can use version number from `style-style.asset.php`, same as for scripts. I'll rewrite this tomorrow.

Reply:
… or maybe not. This file is not generated on `project-base`.

Screenshot from a project:
<img width="204" alt="Screenshot 2021-09-09 at 20 42 16" src="https://user-images.githubusercontent.com/16900754/132744308-876e6ebe-704a-4415-bfac-60c60065cf84.png">

Reply:
@Clorith , @walbo Any suggestions?

Reply:
Use theme version when `WP_ENVIRONMENT_TYPE === production` and `filemtime` in other envs?

Reply:
Reverting to filemtime